src/form.js

```diff
@@ -174,12 +174,16 @@ define([
             var form = this,
                 vellum = form.vellum,
                 oldHashtags = form.hashtagMap;
+            form.hashtagNamespaces = {form: true};
             if (form.richText) {
                 // TODO always load hashtags, even when rich text is disabled
                 form.hashtagMap = _.clone(vellum.datasources.getHashtagMap({}));
                 form.invertedHashtagMap = _.invert(form.hashtagMap);
                 form.hashtagTransformations = vellum.datasources.getHashtagTransforms({});
                 form.hasCaseHashtags = vellum.datasources.isReady();
+                _.each(form.hashtagTransformations, function (x, prefix) {
+                    form.hashtagNamespaces[/^#([^\/]+)/.exec(prefix)[1]] = true;
```

Comment:
b0baecfedf71b358e8ef8a52883e83ffc606fa34

The input keys here are hashtag hashtag transformations, which are always populated from the hashtags supplied in the data sources schema (or in the short term, `#case/` guesses). I don't think there is any way a value like `#user+case` could get here unless it was specified in the schema, in which case we'd likely want it here.